pipelines/h6c/idr2/v3/lstbin/lstbin-template.toml

```diff
@@ -82,14 +82,15 @@ overwrite=true
 write_med_mad=false
 plot_every=180  # about 1 hour
 save_metric_data = true  # this should be updated for the real runs
+save_lstbin_data = true
 do_extra_flagging = false
-do_simultaneous_inpainting = {{ (not FLAGGED_AVERAGE)|string|lower}}
+inpaint_method="per-night"
```

Comment:
shouldn't these have spaces around them>

Reply:
It needn't, but it is nicer with spaces. I'll add them.